zerver/lib/bugdown/__init__.py

```diff
@@ -599,12 +599,30 @@ def is_image(self, url: str) -> bool:
         if not self.image_preview_enabled():
             return False
         parsed_url = urllib.parse.urlparse(url)
+        # remove html urls which end with img extensions that can not be shorted
+        if parsed_url.netloc == 'pasteboard.co':
+            return None
```

Comment:
Why does this return None, not False or True?  I'm surprised mypy didn't flag this as a bug.